lib/internal/url.js

```diff
@@ -1308,11 +1308,17 @@ function originFor(url, base) {
 }
 
 function domainToASCII(domain) {
+  if (arguments.length < 1)
```

Comment:
Can you change these to checks against `domain` explicitly. No need to introduce `arguments` for no reason.